vmdb/spec/models/miq_provision_virt_workflow_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "spec_helper"
+
+describe MiqProvisionVirtWorkflow do
+  context "#continue_request" do
+    let(:sdn) { "SysprepDomainName" }
+    let(:wf)  { FactoryGirl.create(:miq_provision_virt_workflow) }
+
+    before do
+      FactoryGirl.create(:user_admin)
+      MiqDialog.seed
```

Comment:
This is what it's doing (I did MiqRegion.seed; MiqDialog.seed in a rails console with an empty db):

https://gist.github.com/jrafanie/c8e6eec463eeb3e56b4f

I'd prefer to add only the required things, the gist should give you what you need to create only what you need.